Makefile

```diff
@@ -3,9 +3,12 @@
 all: precommit
 
 .PHONY: lint
+## Check the formatting of all files, run clippy on the source code, then run
+## clippy on the tests (but allow expect to be used in tests)
 lint:
 	cargo fmt -- --check
-	cargo clippy --all-targets --all-features -- -D warnings -W clippy::unwrap_used
+	cargo clippy --all-features -- -D warnings -W clippy::unwrap_used -W clippy::expect_used
+	cargo clippy --tests -- -D warnings -W clippy::unwrap_used
```

Comment:
if you are doing `allow` in the test code where we are using `expect`, then you don't need to split these here, do you?

Reply:
I used the `allow` in the test setup and teardown files since those are marked as "not tests" [here](https://github.com/momentohq/client-sdk-rust/blob/main/Cargo.toml#L18-L30), but I didn't want to go in and add the `allow` everywhere we used `expect` in the test files

Reply:
:ack: